src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -244,17 +245,36 @@ void Profile::LayerJson(const Json::Value& json)
 }
 
 // Method Description:
+// - Either Returns this profile's background image path, if one is set, expanding
 // - Returns this profile's background image path, if one is set, expanding
 //   any environment variables in the path, if there are any.
+// - Or if "DesktopWallpaper" is set, then gets the path to the desktops wallpaper.
 // Return Value:
-// - This profile's expanded background image path / the empty string.
+// - This profile's expanded background image path / desktops's wallpaper path /the empty string.
 winrt::hstring Profile::ExpandedBackgroundImagePath() const
 {
     if (_BackgroundImagePath.empty())
     {
         return _BackgroundImagePath;
     }
-    return winrt::hstring{ wil::ExpandEnvironmentStringsW<std::wstring>(_BackgroundImagePath.c_str()) };
+    // checks if the user would like to copy their desktop wallpaper
+    // if so, replaces the path with the desktop wallpaper's path
+    else if (_BackgroundImagePath == to_hstring(DesktopWallpaperEnum))

```

Comment:
If we're just going to be constructing an `hstring` from `DesktopWallpaperEnum` inevitably, shouldn't we just store it as an `hstring`?

Reply:
Yea that makes sense. Resolved.